source/assets/javascripts/app/_download_new.js

```diff
@@ -83,8 +89,9 @@ var newShowDownloadLinks = (function ($) {
         return latestRelease;
       };
 
-      var releases = R.compose(R.sort(compareVersions('go_full_version')), R.map(addURLToFiles), R.map(addDisplayVersion))(releaseData[0]);
-      var amiReleases = R.sort(compareVersions('go_version'))(amiData[0]);
+      var releases = R.compose(releasesLessThanAYearOld, R.sort(compareVersions('go_full_version')), R.map(addURLToFiles), R.map(addDisplayVersion))(releaseData[0]);
+      var amiReleases = R.slice(0, releases.length, R.sort(compareVersions('go_version'))(amiData[0]));
```

Comment:
@arvindsv - I need to update https://download.gocd.org/cloud.json to provide information about the time at which the images were created/updated. So, I thought of putting this till I update the script that updates cloud.json. Cool?

Reply:
Yes, that's fine.